src/wireviz/Harness.py

```diff
@@ -171,15 +169,13 @@ def create_graph(self) -> Graph:
                      f'{cable.gauge} {cable.gauge_unit}{awg_fmt}' if cable.gauge else None,
                      '+ S' if cable.shield else None,
                      f'{cable.length} m' if cable.length > 0 else None,
-                     cable.color, '<!-- colorbar -->' if cable.color else None],
+                     cable.color, html_colorbar(cable.color)],
                     '<!-- wire table -->',
+                    [html_image(cable)],
```

Comment:
`html_image(cable.image, cable.image_scale, cable.image_size)` maybe?

Reply:
It also needs the `cable.caption`, and I see no harm to let the helper function see all common attributes of nodes. In object oriented code, this would have been a method of the node superclass.

Reply:
How about nesting the image attributes in the input YAML file?
```yaml
connectors:
  Key:
    type: Phone Connector
    subtype: male 3.5
    pins: [T, R, S]
    pinlabels: [Dot, Dash, Ground]
    show_pincount: false
    image:
      src: ../resources/stereo-phone-plug-TRS.png
      caption: Tip, Ring, and Sleeve
      width: 100
      height: 100
      scale: both
```

- It keeps all the attributes neatly together
- You only need to pass the `node.image` attribute to the function and have all you need
- results in shorter attributes (`scale`, `width`, ... instead of `image_scale`, `image_size`, ...)

Reply:
Good idea! I like it, and I will look into what changes are needed.

Reply:
I am open to changing `src` to `source`, `file`, `filename` or anything else if you think it's more user friendly... but `src` is also fine by me.